tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetMvc5IastTests.cs

```diff
@@ -116,6 +116,7 @@ public AspNetMvc5IastTests(IisFixture iisFixture, ITestOutputHelper output, bool
         SetEnvironmentVariable("DD_IAST_REQUEST_SAMPLING", "100");
         SetEnvironmentVariable("DD_IAST_MAX_CONCURRENT_REQUESTS", "100");
         SetEnvironmentVariable("DD_IAST_VULNERABILITIES_PER_REQUEST", "100");
+        SetEnvironmentVariable("DD_TRACE_DEBUG", "1");
```

Comment:
Probably shouldn't leave this on?

Reply:
Yes, it'd be better to remove it. Thanks!